internal/pkg/manifest/workload.go

```diff
@@ -443,14 +446,14 @@ type SidecarConfig struct {
 
 // TaskConfig represents the resource boundaries and environment variables for the containers in the task.
 type TaskConfig struct {
-	CPU            *int              `yaml:"cpu"`
-	Memory         *int              `yaml:"memory"`
-	Platform       *string           `yaml:"platform,omitempty"`
-	Count          Count             `yaml:"count"`
-	ExecuteCommand ExecuteCommand    `yaml:"exec"`
-	Variables      map[string]string `yaml:"variables"`
-	Secrets        map[string]string `yaml:"secrets"`
-	Storage        *Storage          `yaml:"storage"`
+	CPU            *int                  `yaml:"cpu"`
+	Memory         *int                  `yaml:"memory"`
+	Platform       *PlatformArgsOrString `yaml:"platform,omitempty"`
```

Comment:
In `AppRunnerInstanceConfig` the type is not a pointer. Is there a reason why there is a difference between the two?

Reply:
changing back to `*string` now that we're not surfacing that field for RDWS